src/dispatch/static/dispatch/src/signal/NewEditDialog.vue

```diff
@@ -100,6 +100,16 @@
                     :rules="[rules.required]"
                   />
                 </v-col>
+                <v-col cols="12">
+                  <v-text-field
+                    v-model="lifecycle"
+                    label="Lifecycle"
+                    hint="The lifecycle stage of the signal."
+                    persistent-hint
+                    name="lifecycle"
+                    :rules="[rules.required]"
```

Comment:
I don't think it should be required if it's nullable, right?

Reply:
(I don't actually know) -- Just worried about the bug where you can't save the edit sheet because of the required rule.

Reply:
This is expected to be populated by Geiger, so it should always have a value, and it shouldn't be edited manually. I could remove `:rules="[rules.required]"` and add `readonly `.

Reply:
I added `readonly` and kept `:rules="[rules.required]"`.

Reply:
I did it locally but didn't push the changes upstream lol. I'll add the changes in a separate PR.